sui_core/src/authority.rs

```diff
@@ -461,46 +461,6 @@ impl AuthorityState {
         })
     }
 
-    /// Process certificates coming from the consensus. It is crucial that this function is only
-    /// called by a single task (ie. the task handling consensus outputs).
```

Comment:
It would be nice not to lose this comment. Even better, it would be great to have this enforced through concurrency primitives -if that's still necessary after the refactor to the lock service. Do we have a task to track installing a mutex on this?
/cc @asonnino

Reply:
There should already be an atomic to entire this (and yes, let's not lose this kind of comments).
https://github.com/MystenLabs/sui/blob/dfe87bcb460b2600299a089e2ca1fab6e6e8117e/sui_core/src/authority.rs#L1155-L1165